### PR TITLE
[FT:] Assert error is in error list

### PR DESCRIPTION
**Pull request by simiyu1**
Resolving TTL feedback

> This is predefined post method that we are using to perform posts in test cases below, should it have an assert? given that it will yield different responses depending on how it's called?

### CODE REVIEW

**Comment by simiyu1** [comment]
Resolves group pull request [Pull request #11](https://github.com/andela/hc-june-bunnies/pull/11)

File: hc/api/tests/test_create_check.py [hunk 1]
```diff
@@ -11,14 +11,17 @@ def setUp(self):
...
+        response = self.client.post(self.URL, json.dumps(data),
                              content_type="application/json")
+        error_list = [ "wrong api_key","could not parse request body","timeout is not a number","name is not a string"]
 
         if expected_error:
-            self.assertEqual(r.status_code, 400)
-            ### Assert that the expected error is the response error
+            self.assertEqual(response.status_code, 400)
+            this_error = response.json()
+            #Asserts that the expected error is the response error
```
Comment (thoth-ky) [inline hc/api/tests/test_create_check.py:21]: Use a space after `#`

Comment (simiyu1) [reply hc/api/tests/test_create_check.py:21]: Resolved

**Review by dmaish — APPROVED** [review]
The comments is the list seem to be working fine.

**Comment by miritih** [comment]
Changes Already merged

**Pull request closed without merging**